README.md

```diff
@@ -230,6 +237,35 @@ class SomeAdminOrFormClass(...):
         }
 ```
 
+## Add vendor prefixes to CSS rules using values from https://caniuse.com/
+
+Writing SCSS shall be fast and easy and you should not have to care, whether to add vendor specific
+prefixes to your CSS directives. Unfortunately there is no pure Python package to solve this, but
+with a few node modules, we can add this to our process chain.
+
+As superuser install
+
+```shell
+npm install -g npx
```

Comment:
Shouldn't we recommend using Yarn instead? Then the command is `yarn global add npx`.

Reply:
I have no opinion on this. I use ``npm`` for all of my projects and had no problems with it up to now. It what respect is ``yarn`` better?

Reply:
I've heard some negative thing related to the development of npm. Besides that yarn also uses more concise lockfiles and checksums, but npm could've also implemented this by now. I'm not a real fan of Javascript, but Yarn seems to be what everyone is transitioning towards. It's also faster in comparison from what I read online.